lib/ts/framework/fastify/framework.ts

```diff
@@ -167,7 +167,7 @@ export const errorHandler = () => {
         let supertokens = SuperTokens.getInstanceOrThrowError();
         let request = new FastifyRequest(req);
         let response = new FastifyResponse(res);
-        supertokens.errorHandler(err, request, response);
+        await supertokens.errorHandler(err, request, response);
```

Comment:
If this throws an error, what will happen?

Reply:
if the middleware is inside a try catch block, the error will go to that catch block. Else the lambda will throw the error and the API gateway return a 500 response